packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1091,12 +1091,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
         this.logger.sendTelemetryEvent({
             eventName: "ContainerLoadStats",
-            containerLoadDataStoreCount: this.dataStores.containerLoadDataStoreCount,
-            referencedDataStoreCount: this.dataStores.referencedDataStoreCount,
-            createContainerRuntimeVersion: this.createContainerMetadata.createContainerRuntimeVersion,
-            createContainerTimestamp: this.createContainerMetadata.createContainerTimestamp,
-            summaryCount: this.createContainerMetadata.summaryCount,
-            summaryFormatVersion: metadata?.summaryFormatVersion,
+            ...this.formMetadata(),
```

Comment:
What about the dataStores stats?

Reply:
it is in there, i just added it, i'm also testing if i can use `...this.formMetadata()` in sendTelemetryEvent